scripts/release.sh

```diff
@@ -148,14 +148,14 @@ function setup_upstream() {
   echo "Remote upstream URL is '${upstream}'"
   if [[ -z "${upstream}" ]]; then
     echo "Setting remote upstream URL to '${REPO_UPSTREAM}'"
-    git remote add upstream ${REPO_UPSTREAM}
+    git remote add upstream "${REPO_UPSTREAM}"
   fi
 }
 
 # Fetch the release branch, so we can check it out.
 function setup_branch() {
   [[ -z "${RELEASE_BRANCH}" ]] && return
-  git fetch ${REPO_UPSTREAM} ${RELEASE_BRANCH}:upstream/${RELEASE_BRANCH}
+  git fetch "${REPO_UPSTREAM}" "${RELEASE_BRANCH}":upstream/"${RELEASE_BRANCH}"
```

Comment:
same double-quote-around-the-group-comment here

Reply:
done